ReactiveCocoa/Swift/Signal.swift

```diff
@@ -135,10 +135,15 @@ public final class Signal<Value, Error: ErrorType> {
 				}
 			}
 		} else {
-			observer(.Interrupted)
+			observer.sendInterrupted()
 			return nil
 		}
 	}
+
+	/// Convenience to avoid lots of extra parentheses everywhere people call observe(). TODO(andy): Document me.
+	public func observe(action: Observer.Action) -> Disposable? {
+		return observe(Observer(action))
+	}
```

Comment:
This method seems like of duplicate of the one in the protocol extension (here)[https://github.com/ReactiveCocoa/ReactiveCocoa/pull/2442/files#diff-0d51c589cd1dd91be8a57346073396e7R171].